observability/monitoring/getting-started/core-platform-monitoring-first-steps.adoc

```diff
@@ -6,8 +6,53 @@ include::_attributes/common-attributes.adoc[]
 
+[id="configuring-core-platform-monitoring-postinstallation-steps_{context}"]
+== Configuring core platform monitoring: Postinstallation steps
 
+After {product-title} is installed, cluster administrators typically configure core platform monitoring to suit their needs.
+These activities include setting up storage and configuring options for Prometheus, Alertmanager, and other monitoring components.
+
+[NOTE]
+====
+By default, in a newly installed {product-title} system, users can query and view collected metrics.
+You need only configure an alert receiver if you want users to receive alert notifications.
+Any other configuration options listed here are optional.
+====
+
+* xref:../../../observability/monitoring/configuring-core-platform-monitoring/before-you-begin.adoc#creating-cluster-monitoring-configmap_before-you-begin[Create the `cluster-monitoring-config` `ConfigMap` object] if it does not exist.
+* xref:../../../observability/monitoring/configuring-core-platform-monitoring/configuring-alerts-and-notifications.adoc#configuring-alert-notifications_configuring-alerts-and-notifications[Configure notifications for default platform alerts] so that Alertmanager can send alerts to an external notification system such as email, Slack, or PagerDuty.
+* For shorter term data retention, xref:../../../observability/monitoring/configuring-core-platform-monitoring/storing-and-recording-data.adoc#configuring-persistent-storage_storing-and-recording-data[configure persistent storage] for Prometheus and Alertmanager to store metrics and alert data.
+Specify the metrics data retention parameters for Prometheus and Thanos Ruler.
++
+[IMPORTANT]
+====
+* In multi-node clusters, you must configure persistent storage for Prometheus, Alertmanager, and Thanos Ruler to ensure high availability.
+
+* By default, in a newly installed {product-title} system, the monitoring `ClusterOperator` resource reports a `PrometheusDataPersistenceNotConfigured` status message to remind you that storage is not configured.
+====
++
+* For longer term data retention, xref:../../../observability/monitoring/configuring-core-platform-monitoring/configuring-metrics.adoc#configuring-remote-write-storage-cpm_configuring-metrics[configure the remote write feature] to enable Prometheus to send ingested metrics to remote systems for storage.
++
+[IMPORTANT]
+====
+Be sure to xref:../../../observability/monitoring/configuring-core-platform-monitoring/configuring-metrics.adoc#adding-cluster-id-labels-to-metrics_configuring-metrics[add cluster ID labels to metrics] for use with your remote write storage configuration.
+====
++
+* xref:../../../observability/monitoring/configuring-core-platform-monitoring/before-you-begin.adoc#granting-users-permissions-for-core-platform-monitoring_before-you-begin[Assign monitoring cluster roles] to any non-administrator users that need to access certain monitoring features.
```

Comment:
```suggestion
* xref:../../../observability/monitoring/configuring-core-platform-monitoring/before-you-begin.adoc#granting-users-permissions-for-core-platform-monitoring_before-you-begin[Grant monitoring cluster roles] to any non-administrator users that need to access certain monitoring features.
```
Consider changing the language here a little so that it more closely matches the procedure that it links to.